client/src/Booking.jsx

```diff
@@ -108,18 +114,16 @@ class Booking extends React.Component {
     // })
     //testing version:
     //make sure timestamps r coming in as dates I can parse
-    let inTime = moment().format();
-    let outTime = moment().format();
+    const inTime = moment().format();
+    const outTime = moment().format();
     return axios.get('http://localhost:3002/booking/bookingTotal', { params: {
       campId: 0,
       check_in_date: inTime,
       check_out_date: outTime
       }
     })
-      .then((res) => {
-        console.log('res: ', res);
-        // this.setState({
-        // });
+      .then(({data}) => {
+        console.log('booking tot. data: ', data);
```

Comment:
🤔 what happens here?

Reply:
Good Q — I need to write code that updates each of these values in my Booking state obj and then use them on the '/booking/bookingTotal' view.
Properties: total_days, average_price_X_nights, subTotal